pint/observatory/topo_obs.py

```diff
@@ -50,8 +51,12 @@ def __init__(self, name, tempo_code=None, itoa_code=None, aliases=None,
                           values).  Default='tempo'
             include_gps = Set False to disable UTC(GPS)->UTC clock
                           correction.
-            include_bipm = Set False to disable TAI TT(BIPM) clock
+            include_bipm= Set False to disable UTC-> tt BIPM clock
```

Comment:
* TT should always be capitalized
* This comment isn't very clear to me. The include_bipm flag sets what is done to convert TAI to TT, right? It has nothing to do with the UTC to TAI conversion. So, this docstring should say that if include_bipm is false, then then it will use TT = TAI+32.184s (Is that correct?  I think that must be what astropy.Time does for the TAI to TT conversion, but please check).  If it is true it will use TT=TT(BIPMYYYY). And, you might also include this hyperlink: http://www.bipm.org/en/bipm-services/timescales/time-ftp/ttbipm.html